package.json

```diff
@@ -205,7 +205,7 @@
   "bundlesize": [
     {
       "path": "./dist/grommet.min.js",
-      "maxSize": "145 kB"
+      "maxSize": "245 kB"
```

Comment:
Seems like we shouldn't need to increase by this much.

Reply:
fixed!